python/tk_framework_desktopserver/server.py

```diff
@@ -21,13 +21,15 @@
 
 from .errors import MissingCertificateError, PortBusyError
 
+from .logger import get_logger
+
 
 class Server:
     _DEFAULT_PORT = 9000
     _DEFAULT_KEYS_PATH = "../resources/keys"
     _DEFAULT_WHITELIST = "*.shotgunstudio.com"
 
-    def __init__(self, port=None, debug=True, whitelist=None, keys_path=None):
+    def __init__(self, port=None, debug=False, whitelist=None, keys_path=None):
```

Comment:
Yeah, as I said the other day during my presentation, the framework needs a LOT of love.